### ISSUE TITLE
False positive (regression) bad-continuation

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#3326]
### Steps to reproduce

file:
```python
if (a and
    b) or c:
    print(d)
```

### Current behavior

```
convention (C0330, bad-continuation, ) Wrong continued indentation before block (add 4 spaces).
     b) or c:
     ^   |
```

### Expected behavior

no diagnostics (as it was before the update this morning)

### pylint --version output

```
pylint 2.4.4
astroid 2.3.3
Python 3.7.6 (default, Dec 30 2019, 19:38:28) 
[Clang 11.0.0 (clang-1100.0.33.16)]
```

**Comment by PCManticore** [comment]
Thanks for the report. I'm not entirely sure we modified `bad-continuation` in recent versions, but that code doesn't look proper according to PEP-8 to me. Regardless, this issue is probably a form of https://github.com/PyCQA/pylint/issues/289 which is a long standing issue with our PEP-8 checks regarding proper indentations.
